crates/s3s-fs/src/s3.rs

```diff
@@ -272,10 +272,10 @@ impl S3 for FileSystem {
             }
 
             let file_meta = try_!(entry.metadata().await);
-            let creation_date = Timestamp::from(try_!(file_meta.created()));
+            let created_or_modified_date = Timestamp::from(try_!(file_meta.created().or(file_meta.modified())));
```

Comment:
You can add a comment referencing this PR. It looks strange at first glance but it is a reasonable hack.

Reply:
Comment added. ✔️